dnf-behave-tests/dnf/steps/lib/dnf.py

```diff
@@ -109,6 +115,16 @@ def find_transaction_table_end(lines):
         match = SEPARATOR_RE.match(line)
         if match:
             return i - 1
+
+    # DNF 5 requires a different match
+    for i in range(0, len(lines)):
+        if not lines[i].strip():
+            # empty line indicates the end of the transaction table
+            return i
+        if lines[i].startswith("Transaction Summary:"):
+            # "Transaction Summary" follows the transaction table
+            return i
```

Comment:
So can both of the above mark the end of the table? It should only be one of them? We should be more strict when parsing the table and not match more than one semi-arbitrary string.

Reply:
All right, let's make it an empty line then.
If `Transaction Summary:` immediately follows the transaction table, we should fix the code by adding the empty line.